sources/3.0.0/sections/10-data_product_format.adoc

```diff
@@ -594,7 +594,7 @@ unsigned +
 | commonPointRule
 | 1
 | Enumeration
-| Value: 1 (average) or other values from <<iho-s100,part-10c,table=20>>.
+| Value: 1-4 <<iho-s100,part-10c,table=20>>.
```

Comment:
A thought: should this be fixed to value 2 ("Use the least of the attribute values")?
I think this might be sensible, since we're talking about a navigation product. Value 2 would keep things on the safe side.
Maybe this is a question that should be posted as an issue for discussion?

Reply:
SMA strongly support value 2. However, there might be others who prefer the other alternatives. As you say, a new issue might be suitable.

Reply:
Posted a new issue: https://github.com/iho-ohi/S-102-Product-Specification/issues/74

Reply:
I beg to differ.
You can query a grid at exactly at border of two (or three or four) grid cells containing data. If this happens, you have several nearest neighbours, so which one to report? I think the shoalest value makes most sense.

Reply:
S-102PT16 decided to limit the allowed value to "2: low". This change has been made and is included in the pull request.
I'll mark this conversation as resolved.